pkg/bootstrap/bootstrap_config.go

```diff
@@ -211,6 +211,25 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 		}
 		StoreHostPort(h, p, "zipkin", opts)
 	}
+	if config.LightstepAddress != "" {
+		h, p, err = GetHostPort("Lightstep", config.LightstepAddress)
+		if err != nil {
+			return "", err
+		}
+		StoreHostPort(h, p, "lightstep", opts)
+	}
+	if config.LightstepAccessTokenFile != "" {
+		lightstepAccessTokenPath := path.Join(config.ConfigPath, "lightstep_access_token")
+		lsConfigOut, err := os.Create(lightstepAccessTokenPath)
```

Comment:
This was a hack because I was struggling to figure out how to take an externally mounted file with our access token and mount it into the envoy container. so i cheated and put the token in plain text and wrote it into a file inside the already mounted configmap.